pkg/api/validation/validation_test.go

```diff
@@ -714,8 +742,37 @@ func TestValidatePersistentVolumeClaim(t *testing.T) {
 				StorageClassName: &invalidClassName,
 			}),
 		},
+		"invalid-volumemode": {
+			isExpectedFailure: true,
+			claim: testVolumeClaim("foo", "ns", api.PersistentVolumeClaimSpec{
+				Selector: &metav1.LabelSelector{
+					MatchExpressions: []metav1.LabelSelectorRequirement{
+						{
+							Key:      "key2",
+							Operator: "Exists",
+						},
+					},
+				},
+				AccessModes: []api.PersistentVolumeAccessMode{
+					api.ReadWriteOnce,
+					api.ReadOnlyMany,
+				},
+				Resources: api.ResourceRequirements{
+					Requests: api.ResourceList{
+						api.ResourceName(api.ResourceStorage): resource.MustParse("10G"),
+					},
+				},
+				StorageClassName: &validClassName,
+				VolumeMode:       &invalidMode,
+			}),
+		},
 	}
 
+	err := utilfeature.DefaultFeatureGate.Set("BlockVolumeSupport=true")
```

Comment:
We shouldn't enable the alpha feature in this suite. Instead, we can have a test case here to validate you can't specify PVC.volumeType.